modules/metastorage-client/src/integrationTest/java/org/apache/ignite/internal/metastorage/client/ITMetaStorageServiceTest.java

```diff
@@ -202,7 +202,7 @@ public void beforeTest() throws Exception {
                 addr -> ClusterServiceTestUtils.clusterService(
                     addr.toString(),
                     addr.port(),
-                    nodeFinder,
+                    LocalPortRangeNodeFinder.createConfiguration(NODE_PORT_BASE, NODE_PORT_BASE + NODES),
```

Comment:
ok, this looks weird. Can a NodeFinder have a non-static method, called "updateConfiguration", that accepts a `NodeFinderChange` instead?